x-pack/plugins/security_solution/public/flyout/entity_details/host_right/fields/endpoint_policy_fields.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiHealth } from '@elastic/eui';
+
+import type { EntityTableRows } from '../../shared/components/entity_table/types';
+import type { ObservedEntityData } from '../../shared/components/observed_entity/types';
+import { EndpointAgentStatus } from '../../../../common/components/endpoint/endpoint_agent_status';
+import { getEmptyTagValue } from '../../../../common/components/empty_value';
+import type { HostItem } from '../../../../../common/search_strategy';
+import { HostPolicyResponseActionStatus } from '../../../../../common/search_strategy';
+import * as i18n from './translations';
+
+export const policyFields: EntityTableRows<ObservedEntityData<HostItem>> = [
+  {
+    label: i18n.ENDPOINT_POLICY,
+    render: (hostData: ObservedEntityData<HostItem>) => {
+      const appliedPolicy = hostData.details.endpoint?.hostInfo?.metadata.Endpoint.policy.applied;
+      return appliedPolicy?.name ? <>{appliedPolicy.name}</> : getEmptyTagValue();
+    },
+    isVisible: (hostData: ObservedEntityData<HostItem>) => hostData.details.endpoint != null,
+  },
+  {
+    label: i18n.POLICY_STATUS,
+    render: (hostData: ObservedEntityData<HostItem>) => {
+      const appliedPolicy = hostData.details.endpoint?.hostInfo?.metadata.Endpoint.policy.applied;
+      return appliedPolicy?.status ? (
+        <EuiHealth
```

Comment:
NIT: with the ternary on the color prop, this check kinda confused me at first
Maybe we can assign the element to a var and just use that on the return ternary? might make it more readable?